mapiproxy/servers/default/emsmdb/oxcfxics.c

```diff
@@ -3353,8 +3403,27 @@ _PUBLIC_ enum MAPISTATUS EcDoRpc_RopSyncImportReadStateChanges(TALLOC_CTX *mem_c
 
 	request = &mapi_req->u.mapi_SyncImportReadStateChanges;
 
+	retval = openchangedb_get_MailboxReplica(emsmdbp_ctx->oc_ctx, emsmdbp_get_owner(synccontext_object), NULL, &replica_guid);
+	if (retval != MAPI_E_SUCCESS) {
+		OC_DEBUG(5, "Impossible to get %s mailbox replica guid", emsmdbp_get_owner(synccontext_object));
+		mapi_repl->error_code = MAPI_E_CALL_FAILED;
```

Comment:
:question: returning MAPI_E_CALL_FAILED makes sense to me, but would be good if we could ensure that vs Outlook. Would you any way to test this easily and see if it causes Outlook to produce a sync issue or if it is OK with that and move on?